functions/Add-DbaAgDatabase.ps1

```diff
@@ -174,6 +174,17 @@ function Add-DbaAgDatabase {
                     Stop-Function -Continue -Message "Could not connect to instance $($secondaryInstance.Name)"
                 }
 
+                if ($SeedingMode -eq "Automatic" -and $secondaryInstance.VersionMajor -le 12)
+                {
+                    Stop-Function -Continue -Message "Automatic seeding mode not supported on SQL Server versions prior to 2016 - Instance $($secondaryInstance.Name)"
+
+                    if (!($SharedPath) -and $UseLastBackup -eq $false)
+                    {
+                        Stop-Function -Continue -Message "Automatic seeding not supported and no $SharedPath provided. Database not added to instance $($secondaryInstance.Name)"
+                        break
```

Comment:
the `break` will never be reached because of the `-Continue` above it. if you need to stop the function, please use return in place of break and remove Continue.

Reply:
I only need to break the loop this code is in (foreach), not the whole function. What's the best way to do that?

Reply:
-Continue should do the trick. Please test, however! but I was told by fred pretty much never to break, so i imagine it'd be a continue either way.